README.md

```diff
@@ -32,28 +32,28 @@ When we are coding, we have to be sure that our code is working as we expect. Mo
 
 We need to execute our tests. First of all, fork and clone this repo into your Github account. Once you are done, open the file `SpecRunner.html` with your browser.
 
-In the beginning, you will see all the tests in yellow because the tests we have to implement are commented. (except for one, that througs an error, no worries :wink:)
+In the beginning, you will see all the tests in yellow because the tests we have to implement are commented. (except for one, that throws an error, no worries :wink:)
 
-![image](https://user-images.githubusercontent.com/23629340/36485165-c51a20fc-171b-11e8-854c-a83fb7dec036.png)
+![image](http://materials.ironhack.com/s/HkAOp3cvQ)
```

Comment:
This is the URL of note but you want the URL of the image file. You can find it inside the markdown code of the editor. 
![screen shot 2018-09-06 at 17 33 55](https://user-images.githubusercontent.com/621544/45168255-1b6f8080-b1fb-11e8-9eab-35eac03a3e0d.png)